src/components/Story/Story.js

```diff
@@ -30,19 +31,39 @@ function Story() {
 
       <Switch>
         <Route exact path="/story">
-          <NavLink to={`/story/${dayNum}/day`}>
-            <button>Continue</button>
-          </NavLink>
+          {dayNum < 5 && (
+            <NavLink to={`/story/${dayNum}/day`}>
+              <button>Continue</button>
```

Comment:
Should we consider removing the logic from the return for both dayNum conditions and storing it in a var above the return? Something like this above the return:

```
let storyProgress;
 if(dayNum < 5){
      storyProgress  = <NavLink to={`/story/${dayNum}/day`}>
                                      <button>Continue</button>
                                   </NavLink>
 } else if (dayNum > 5) {
      storyProgress  =  <>
                                       <p className="finale-container">
                                         Wow what a week! Frankly, youre exhausted and have no idea what
                                         to expect next. As you get out of bed and head into the kitchen
                                         you see Fluffy already sitting there, waiting for you. It looks
                                         like they have something important to say...
                                       </p>

                                      <NavLink to={`/story/finale/`}>
                                          <button>Let's find out!</button>
                                     </NavLink>
                                  </>
 } 
```

Something like this above the and then we can have just this in the return:
`{storyProgress}`